e2e/tests/ui/features/@sbom-explorer/sbom-explorer.step.ts

```diff
@@ -12,8 +12,14 @@ const VULN_TABLE_NAME = "Vulnerability table";
 const SBOM_TABLE_NAME = "sbom-table";
 
 Given("An ingested SBOM {string} is available", async ({ page }, sbomName) => {
-  const searchPage = new SearchPage(page, "SBOMs");
-  await searchPage.dedicatedSearch(sbomName);
+  const sbomListPage = await SbomListPage.build(page);
+
+  const toolbar = await sbomListPage.getToolbar();
+  const table = await sbomListPage.getTable();
+
+  await toolbar.applyTextFilter("Filter text", sbomName);
+  await table.waitUntilDataIsLoaded();
+  await table.verifyColumnContainsText("Name", sbomName);
```

Comment:
Instead of using the SearchPage, we can reuse the `SbomListPage` reusable class. 

Although there are more lines involved, `SbomListPage` contains composable components that allow us to interact with specific parts of the Page like `toolbar, table,etc`.

Composable components allow us to create higher blocks of code that can be reused in other places.